### DISCUSSION TITLE
[blocks]:Video Player

### DISCUSSION THREAD

**Discussion by 4ware** [shadcn-ui/ui#3212] [Blocks Request]
### Description

I would love to have a youtube-likee player with controls, progress and possibly chapters,

### Example

https//youtube.com

**Comment by Stem-LG** [comment]
a video player is nice

used [vimejs](https://vimejs.com/) before for custom video players, but has problems such as reliance on web-components (needs dynamic imports in next)

would love something like it with shadcn style and next js support
